uprotocol/core/usubscription/v3/usubscription.proto

```diff
@@ -118,6 +118,16 @@ message SubscribeAttributes {
   // Any additional producer specific information that the consumer must send
   // inside of a subscription request
   repeated google.protobuf.Any details = 2;
+
+  // Maximum Frequency (measured in Hertz) the subscriber wishes to receive events 
+  // for remote only topics. Device dispatchers (i.e. streamers) use this
+  // attribute to reduce the publication rates of events sent between devices.
+  // This attribute is commonly used for mobile/cloud subscribing to vehicle topics
+  // that are published at a high frequency.<br>
+  // If the frequency set by the subscriber is higher than the original publishers
+  // publication frequency, the attribute is ignored.<br>
+  // **__NOTE:__** If this attribute is missing, the frequency is set by the publisher.
+  optional double max_frequency = 3;
```

Comment:
Based on the feedback, seems like this could morph to:
```protobuf
  // Desired Sampling Period (measured in milliseconds) the subscriber wishes to receive events 
  // for remote only topics. Device dispatchers (i.e. streamers) use this
  // attribute to reduce the publication rates of events sent between devices.
  // This attribute is commonly used for mobile/cloud subscribing to vehicle topics
  // that are published at a high rate.<br>
  // If the desired sampling period set by the subscriber is lower than the original publisher's
  // publication period, the attribute is ignored.<br>
  // **__NOTE:__** If this attribute is missing, the sampling period is set by the publisher.
  optional uint32 desired_sampling_period = 3;
```

Reply:
good for me!

Reply:
Thanks peter, I'll change a little but then merge, I'll need +1 from a committer to merge.

Reply:
Just a point of curiosity for me -- why would we choose an int32 over a uint32 if we know that the value will always be unsigned?